app/styles/app.scss

```diff
@@ -154,6 +154,47 @@
   width: 100%;
 }
 
+
+.btn-filter {
+  background-color: Transparent;
+  background-repeat:no-repeat;
+  cursor:pointer;
+  overflow: hidden;
+  outline:none;
+}
+
+.panel-body {
```

Comment:
You'll want to avoid using generic names if this styling is for a specific element or page.

Using SCSS, you can make it a bit less verbose by using nesting

```scss
.review-list {
    heading {
        ....
    }
}
```